Added math.log2 and hypot

Chillee opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#21512 Added math.log2 and hypot**
* #21511 Refactored math tests to iterate over all math ops

Differential Revision: [D16088191](https://our.internmc.facebook.com/intern/diff/D16088191)

Chillee committed:
Added math.log2 and hypot

Chillee committed:
Update on "Added math.log2 and hypot"

Added math.log2 and hypot

gh-metadata: pytorch pytorch 21512 gh/chillee/36/head

Chillee committed:
Update on "Added math.log2 and hypot"

Added math.log2 and hypot

gh-metadata: pytorch pytorch 21512 gh/chillee/36/head

Chillee committed:
Update on "Added math.log2 and hypot"

Added math.log2 and hypot

gh-metadata: pytorch pytorch 21512 gh/chillee/36/head

Chillee committed:
Update on "Added math.log2 and hypot"

Added math.log2 and hypot

gh-metadata: pytorch pytorch 21512 gh/chillee/36/head

Chillee committed:
Update on "Added math.log2 and hypot"

Added math.log2 and hypot

gh-metadata: pytorch pytorch 21512 gh/chillee/36/head

Chillee committed:
Update on "Added math.log2 and hypot"

Added math.log2 and hypot

gh-metadata: pytorch pytorch 21512 gh/chillee/36/head

Chillee committed:
Update on "Added math.log2 and hypot"

Added math.log2 and hypot

gh-metadata: pytorch pytorch 21512 gh/chillee/36/head

Chillee committed:
Update on "Added math.log2 and hypot"

Added math.log2 and hypot

gh-metadata: pytorch pytorch 21512 gh/chillee/36/head

Chillee committed:
Update on "Added math.log2 and hypot"

Added math.log2 and hypot

gh-metadata: pytorch pytorch 21512 gh/chillee/36/head

Chillee committed:
Update on "Added math.log2 and hypot"

Added math.log2 and hypot

gh-metadata: pytorch pytorch 21512 gh/chillee/36/head

Chillee committed:
Update on "Added math.log2 and hypot"

Added math.log2 and hypot

gh-metadata: pytorch pytorch 21512 gh/chillee/36/head

jamesr66a reviewed (approved):
nice

ailzhang commented:
Oh in fact, you might need to rebase this on top of master before landing. I think I used `hypot` as the one to test out "unsupported function" error message. And that got merged 1-2 days ago. #21068 You might want to change it to another rarely used function instead. Thanks!

Chillee committed:
Update on "Added math.log2 and hypot"

Added math.log2 and hypot

gh-metadata: pytorch pytorch 21512 gh/chillee/36/head

Chillee committed:
Update on "Added math.log2 and hypot"

Added math.log2 and hypot

gh-metadata: pytorch pytorch 21512 gh/chillee/36/head

Chillee committed:
Update on "Added math.log2 and hypot"

Added math.log2 and hypot

gh-metadata: pytorch pytorch 21512 gh/chillee/36/head

Chillee committed:
Update on "Added math.log2 and hypot"

Added math.log2 and hypot

gh-metadata: pytorch pytorch 21512 gh/chillee/36/head

Chillee committed:
Update on "Added math.log2 and hypot"

Added math.log2 and hypot

gh-metadata: pytorch pytorch 21512 gh/chillee/36/head

Chillee committed:
Update on "Added math.log2 and hypot"

Added math.log2 and hypot

gh-metadata: pytorch pytorch 21512 gh/chillee/36/head

Chillee committed:
Update on "Added math.log2 and hypot"

Added math.log2 and hypot

gh-metadata: pytorch pytorch 21512 gh/chillee/36/head

Chillee committed:
Update on "Added math.log2 and hypot"

Added math.log2 and hypot

gh-metadata: pytorch pytorch 21512 gh/chillee/36/head

Chillee committed:
Update on "Added math.log2 and hypot"

Added math.log2 and hypot

gh-metadata: pytorch pytorch 21512 gh/chillee/36/head

Chillee committed:
Update on "Added math.log2 and hypot"

Added math.log2 and hypot

gh-metadata: pytorch pytorch 21512 gh/chillee/36/head

Chillee committed:
Update on "Added math.log2 and hypot"

Added math.log2 and hypot

gh-metadata: pytorch pytorch 21512 gh/chillee/36/head
